alibi/explainers/similarity/backends/tensorflow/base.py

```diff
@@ -79,3 +98,11 @@ def argmax(X: tf.Tensor, dim=-1) -> tf.Tensor:
         """Returns the index of the maximum value in a tensor."""
         X = tf.math.argmax(X, axis=dim)
         return X
+
+    @staticmethod
+    def get_non_trainable(model: keras.Model) -> List[Union[int, str]]:
+        """Checks if all layers in a model are trainable.
+
+        Note: batch normalization layers are ignored as they are not trainable by default.
```

Comment:
I don't understand the note. Batch normalization has some trainable parameters. Can you clarify what 'are ignored' means in this context?

Reply:
Marking resolved as we discussed offline and agreed to no longer filter for non_trainable weights set by default like in batch norm layers. Instead, we just warn the user by printing a list of such parameters.